spec/integration/network/http/api/indirected_routes_spec.rb

```diff
@@ -2,6 +2,8 @@
 require 'spec_helper'
 require 'puppet/network/http'
 require 'puppet/network/http/api/indirected_routes'
+require 'rack/mock' if Puppet.features.rack?
```

Comment:
The `if` in this line is surprising to my 👀  although I claim no real familiarity here.  Would it be more conventional to push the `require 'rack/mock'` down into the test case which actually needs it?